src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -5,8 +5,8 @@
 
 # TODO Move this to a package shared by CLI and SDK
 from functools import total_ordering
-from importlib import import_module
 from enum import Enum
+from importlib import import_module  # pylint: disable=wrong-import-order
```

Comment:
😆 🤣 I know right!?  

It's because Python 2 thinks they should be one way and Python 3 thinks they should be the other.  Literally there is no way to satisfy them both.  Gooooood tiiiiimes.